packages/release/label-data-alternative.service

```diff
@@ -0,0 +1,19 @@
+[Unit]
+Description=Label alternative data partition
+DefaultDependencies=no
+Conflicts=shutdown.target
+# Only run this if a partition labeled 'BOTTLEROCKET-DATA' does not exist already.
+ConditionPathIsSymbolicLink=!/dev/disk/by-partlabel/BOTTLEROCKET-DATA
+# This is the partition GUID for the fallback data partition.
+Wants=dev-disk-by\x2dpartuuid-69040874\x2d417d\x2d4e26\x2da764\x2d7885f22007ea.device
+After=dev-disk-by\x2dpartuuid-69040874\x2d417d\x2d4e26\x2da764\x2d7885f22007ea.device
+
+[Service]
+Type=oneshot
+
+ExecStartPre=-/usr/bin/sleep 30
```

Comment:
What is this sleep for?

Reply:
It's to give `label-data-preferred` some time to wait for the preferred data partition and label it. If 30 seconds has passed and the preferred data partition doesn't exist either because the EBS volume isn't attached then we fallback to the alternative data partition with `label-data-alternative`. I kinda touch on this in the commit message as well.

I can add a comment above this line to explain it.